### PR TITLE
[RFC] Check script

### PR DESCRIPTION
**Pull request by drewvolz**
command | description
---|---
` check` | prettier, flow, lint, and test

### CODE REVIEW

**Comment by hawkrives** [comment]
Yes to check, no to the other two.

**Comment by drewvolz** [comment]
What's your reasoning?

**Comment by hawkrives** [comment]
Those are magical incantations. 

My real reasoning is something along the lines of, I have not needed to run them within the span of my memory. 

I don't see a reason to add them.

**Comment by drewvolz** [comment]
Alright... I'll take out the cleaning scripts.

**Comment by hawkrives** [comment]
If you often use the cleaning script, we can add them. I still think you don't need to run them as often as you think you do.

I will support adding _one_ of the cleaning scripts.

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "bundle-data": "node scripts/bundle-data.js data/ docs/",
     "bundle:android": "react-native bundle --entry-file index.android.js --dev true --platform android --bundle-output ./android/app/src/main/assets/index.android.bundle --assets-dest ./android/app/src/main/res/",
     "bundle:ios": "react-native bundle --entry-file index.ios.js --dev true --platform ios --bundle-output ./ios/AllAboutOlaf/main.jsbundle --assets-dest ./ios",
+    "check:": "npm run prettier && npm run flow && npm run lint && npm run test",
```
Comment (hawkrives) [inline package.json]: We don't need to run both "npm run prettier" and "npm run lint" – eslint is run as part of the prettier command.

Comment (drewvolz) [reply package.json]: I'll change that

**Review by hawkrives — APPROVED** [review]

**Pull request merged**